applications/MeshMovingApplication/python_scripts/ale_fluid_solver.py

```diff
@@ -0,0 +1,202 @@
+
+    def ImportModelPart(self):
+        self.fluid_solver.ImportModelPart() # only ONE solver imports the ModelPart
+
+    def PrepareModelPart(self):
+        # Doing it ONLY for the fluid solver (since this contains filling the buffer)
+        return self.fluid_solver.PrepareModelPart()
+
+    def AdvanceInTime(self, current_time):
+        # Doing it ONLY for the fluid solver
+        return self.fluid_solver.AdvanceInTime(current_time)
+
+    def Finalize(self):
+        self.mesh_motion_solver.Finalize()
+        self.fluid_solver.Finalize()
+
+    def InitializeSolutionStep(self):
+        self.mesh_motion_solver.InitializeSolutionStep()
+        self.fluid_solver.InitializeSolutionStep()
+
+    def Predict(self):
+        self.mesh_motion_solver.Predict()
+        self.fluid_solver.Predict()
+
+    def FinalizeSolutionStep(self):
+        self.mesh_motion_solver.FinalizeSolutionStep()
+        self.fluid_solver.FinalizeSolutionStep()
+
+    def SolveSolutionStep(self):
+        self.mesh_motion_solver.SolveSolutionStep()
+
+        # Copy the MESH_VELOCITY to the VELOCITY (ALE) on the interface
+        mesh_vel_components = [
+            KratosMultiphysics.MESH_VELOCITY_X,
+            KratosMultiphysics.MESH_VELOCITY_Y,
+            KratosMultiphysics.MESH_VELOCITY_Z
+        ]
+        vel_components = [
+            KratosMultiphysics.VELOCITY_X,
+            KratosMultiphysics.VELOCITY_Y,
+            KratosMultiphysics.VELOCITY_Z
+        ]
+
+        for i in range(3):
+            for mp in self.ale_interface_parts_by_components[i]:
+                KratosMultiphysics.VariableUtils().CopyScalarVar(
+                    mesh_vel_components[i],
+                    vel_components[i],
+                    mp.Nodes)
+
+        self.fluid_solver.SolveSolutionStep()
+
+    def Check(self):
+        self.mesh_motion_solver.Check()
+        self.fluid_solver.Check()
+
+    def Clear(self):
+        self.mesh_motion_solver.Clear()
+        self.fluid_solver.Clear()
+
+    def GetComputingModelPart(self):
+        return self.mesh_motion_solver.GetComputingModelPart() # this is the same as the one used in Fluid
+
+    def GetFluidSolver(self):
+        return self.fluid_solver
+
+    def GetMeshMotionSolver(self):
+        return self.mesh_motion_solver
+
+    def MoveMesh(self):
+        self.GetMeshMotionSolver().MoveMesh()
+
+
+    def _CreateFluidSolver(self, custom_settings):
+        '''This function creates the fluid solver.
+        It can be overridden to create different fluid solvers
+        '''
+        KratosMultiphysics.CheckRegisteredApplications("FluidDynamicsApplication")
+        import python_solvers_wrapper_fluid
+        fluid_settings = custom_settings.Clone()
+        # remove the ale_settings so we can use the fluid_solver_wrapper constructor
+        fluid_settings["solver_settings"].RemoveValue("ale_settings")
+        return python_solvers_wrapper_fluid.CreateSolver(self.model, fluid_settings)
```

Comment:
I strongly suggest using the equivalent of 

https://github.com/KratosMultiphysics/Kratos/blob/747ef45893c2865541b04c50eaa5425b82080e97/applications/FluidDynamicsApplication/python_scripts/python_solvers_wrapper_fluid.py#L5

since you need to be able to combine solvers. Actually i do think we need to rework the mechanism of the solver_wrappers!!